src/Modules/Content/Dnn.PersonaBar.Recyclebin/Module.build

```diff
@@ -5,6 +5,7 @@
       <DNNFileName>Dnn.PersonaBar.Recyclebin</DNNFileName>
       <PackageName>Dnn.PersonaBar.Recyclebin</PackageName>
       <ModuleFolderName>$(WebsitePath)\DesktopModules\admin\Dnn.PersonaBar\Modules\Dnn.Recyclebin</ModuleFolderName>
+	  <YarnWorkingDirectory></YarnWorkingDirectory>
```

Comment:
@ohine should this have a value?

Reply:
Some of the packages do not have the packages.json file which yarn is looking for, I was having a difficult time getting it to check if the file exists with the wildcard search, so my fix was to check the length of the working directory property. If it's empty Yarn doesn't try to run on that project.